src/modules/roc_pipeline/config.h

```diff
@@ -122,6 +122,9 @@ struct SessionConfig {
 
 //! Receiver parameters.
 struct ReceiverConfig {
+    //! Parameters for the port from which repair packets are sent.
+    PortConfig repair_port;
```

Comment:
Receiver can have an arbitrary number of source and repair ports. That's why we don't have PortConfig is ReceiverConfig. Instead, ports are added dynamically using add_port().

Reply:
455a825301accba5d3f79c7d45ef4315171d0a3f